bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/local/TelldusLocalBridgeHandler.java

```diff
@@ -0,0 +1,313 @@
+        if (newList.getSensors() != null) {
+            if (previouslist == null) {
+                logger.debug("First update of sensors");
+                this.sensorList = newList;
+                for (TellstickLocalSensor sensor : sensorList.getSensors()) {
+                    sensor.setUpdated(true);
+                    for (DeviceStatusListener listener : deviceStatusListeners) {
+                        listener.onDeviceAdded(getThing(), sensor);
+                    }
+                }
+            } else {
+                logger.debug("Update sensors, reset updated flag");
+                for (TellstickLocalSensor sensor : previouslist.getSensors()) {
+                    sensor.setUpdated(false);
+                }
+                logger.debug("Update sensors, reset updated flag1");
+
+                for (TellstickLocalSensor sensor : newList.getSensors()) {
+                    int index = this.sensorList.getSensors().indexOf(sensor);
+                    if (index >= 0) {
+                        TellstickLocalSensor orgSensor = this.sensorList.getSensors().get(index);
+                        logger.debug("Update for sensor:{}", sensor);
+                        orgSensor.setData(sensor.getData());
+                        orgSensor.setUpdated(true);
+                        sensor.setUpdated(true);
+                    } else {
+                        logger.debug("Adding sensor {}", sensor.getId());
+                        this.sensorList.getSensors().add(sensor);
+                        sensor.setUpdated(true);
+                        for (DeviceStatusListener listener : deviceStatusListeners) {
+                            listener.onDeviceAdded(getThing(), sensor);
+                        }
+                    }
+                }
+            }
+            for (TellstickLocalSensor sensor : sensorList.getSensors()) {
+                if (sensor.getData() != null && sensor.isUpdated()) {
+                    for (DeviceStatusListener listener : deviceStatusListeners) {
+                        for (LocalDataTypeValue type : sensor.getData()) {
+                            listener.onDeviceStateChanged(getThing(), sensor,
+                                    new TellstickLocalSensorEvent(sensor.getId(), type.getValue(), type,
+                                            sensor.getProtocol(), sensor.getModel(), System.currentTimeMillis()));
+                        }
+                    }
+                    sensor.setUpdated(false);
+                }
+            }
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            scheduleImmediateRefresh();
+        }
+    }
+
+    @Override
+    public void handleRemoval() {
+        super.handleRemoval();
+    }
+
+    @Override
+    public void thingUpdated(Thing thing) {
+        super.thingUpdated(thing);
+    }
```

Comment:
What's the purpose of these?

Reply:
Not much it looks like, I'll remove them!